pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -908,7 +908,7 @@ func configureLocalDiskToMigrate(dom *libvirtxml.Domain, vmi *v1.VirtualMachineI
 		if _, ok := blockSrcFsDstVols[name]; ok {
 			log.Log.V(2).Infof("Replace block source with destination for volume %s", name)
 			dom.Devices.Disks[i].Source.File = &libvirtxml.DomainDiskSourceFile{
-				File: hostdisk.GetMountedHostDiskDir(name),
+				File: filepath.Join(hostdisk.GetMountedHostDiskDir(name), "disk.img"),
```

Comment:
Nice catch! but shouldn't the PVC mount itself allow O_DIRECT? or maybe I'm not understanding properly?
This is about the path being `/launcherpvcmount` instead of `/launcherpvcmount/disk.img`?

Reply:
ah okay so sort of an undefined territory to open a directory with O_DIRECT

Reply:
apparently it gets en EINVAL :D